ush/diag_statistics.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import os
+import numpy as np
+import pandas as pd
+from netCDF4 import Dataset
+
+data = os.getenv('DATA')
+pdy = os.getenv('PDY')
+cyc = os.getenv('cyc')
+comout = os.getenv('COM_ATMOS_ANALYSIS')
+arcdir = os.getenv('ARCDIR')
+
+
+def get_diag_stats():
+
+    variables = ['sst', 't']
+
+    tarfilename = 'gdas.t' + cyc + 'z.cnvstat'
+    tarfile = os.path.join(comout, tarfilename)
+
+    os.chdir(data)
+
+    for var in variables:
+
+        diagfilename = 'diag_conv_' + var + '_ges.' + pdy + cyc + '.nc4'
+        diagfile = os.path.join(data, diagfilename)
+        zipfilename = diagfilename + '.gz'
+        zipfile = os.path.join(data, zipfilename)
+        outfile = 'cnvstat.' + var + '.gdas.' + pdy + cyc
+
+        # TODO: these probably should be in the jjob. Will try to get them there
+        # once this is up and running
+        command = 'tar -xvf ' + tarfile + ' ' + zipfilename
+        print('running', command)
+        os.system(command)
+        command = 'gunzip ' + os.path.join(data, zipfile)
+        print('running', command)
+        os.system(command)
+
+        # The following is lifted from PyGSI/src/pyGSI/diags.py
+
+        df_dict = {}
+
+        # Open netCDF file, store data into dictionary
+        with Dataset(diagfile, mode='r') as f:
```

Comment:
is a lot of this redundant to what is available for use in PyGSI?

Reply:
PyGSI has more than I really needed from this